src/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs

```diff
@@ -157,30 +534,164 @@ public partial class TypeConverter
+        public bool GetStandardValuesExclusive() { return default(bool); }
+        public virtual bool GetStandardValuesExclusive(System.ComponentModel.ITypeDescriptorContext context) { return default(bool); }
+        public bool GetStandardValuesSupported() { return default(bool); }
+        public virtual bool GetStandardValuesSupported(System.ComponentModel.ITypeDescriptorContext context) { return default(bool); }
+        public virtual bool IsValid(System.ComponentModel.ITypeDescriptorContext context, object value) { return default(bool); }
+        public bool IsValid(object value) { return default(bool); }
+        protected System.ComponentModel.PropertyDescriptorCollection SortProperties(System.ComponentModel.PropertyDescriptorCollection props, string[] names) { return default(System.ComponentModel.PropertyDescriptorCollection); }
+        protected abstract partial class SimplePropertyDescriptor : System.ComponentModel.PropertyDescriptor
+        {
+            protected SimplePropertyDescriptor(System.Type componentType, string name, System.Type propertyType) : base(default(System.ComponentModel.MemberDescriptor)) { }
+            protected SimplePropertyDescriptor(System.Type componentType, string name, System.Type propertyType, System.Attribute[] attributes) : base(default(System.ComponentModel.MemberDescriptor)) { }
+            public override System.Type ComponentType { get; }
+            public override bool IsReadOnly { get; }
+            public override System.Type PropertyType { get; }
+            public override bool CanResetValue(object component) { return default(bool); }
+            public override void ResetValue(object component) { }
+            public override bool ShouldSerializeValue(object component) { return default(bool); }
+        }
+        public partial class StandardValuesCollection : System.Collections.ICollection, System.Collections.IEnumerable
+        {
+            public StandardValuesCollection(System.Collections.ICollection values) { }
+            public int Count { get; }
+            int System.Collections.ICollection.Count { get; }
+            bool System.Collections.ICollection.IsSynchronized { get; }
+            object System.Collections.ICollection.SyncRoot { get; }
+            public object this[int index] { get { return default(object); } }
+            public void CopyTo(System.Array array, int index) { }
+            public System.Collections.IEnumerator GetEnumerator() { return default(System.Collections.IEnumerator); }
+            void System.Collections.ICollection.CopyTo(System.Array array, int index) { }
+            System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { return default(System.Collections.IEnumerator); }
```

Comment:
Same question: why have both public/implicit and private/explicit `GetEnunerator` and `CopyTo` methods?